examples/src/lib.rs

```diff
@@ -62,25 +62,16 @@ pub fn test_example(example: Example, fail: bool) {
         expected_result,
     } = example;
 
-    let options = ProofOptions::new(
-        32,
-        16,
-        0,
-        miden::HashFunction::Blake3_256,
-        miden::FieldExtension::None,
-        8,
-        256,
-    );
-
-    let (mut outputs, proof) = miden::execute(&program, &inputs, num_outputs, &options).unwrap();
+    let (mut outputs, proof) =
+        miden::execute(&program, &inputs, num_outputs, &ProofOptions::default()).unwrap();
 
     assert_eq!(
         expected_result, outputs,
         "Program result was computed incorrectly"
     );
 
     if fail {
-        outputs[0] = outputs[0] + 1;
+        outputs[0] += outputs[0];
```

Comment:
I think this changes the semantics somewhat and if the top of the stack for a given example is 0, this will not result in the failed output. Did you mean to say `output[0] += 1;` here?

Reply:
Oof, yes, that is definitely a typo, sorry! I was just trying to get rid of the clippy warning. It's fixed now.